internal/pkg/manifest/svc.go

```diff
@@ -365,57 +365,23 @@ type HTTPHealthCheckArgs struct {
 	GracePeriod        *time.Duration `yaml:"grace_period"`
 }
 
-func (h *HTTPHealthCheckArgs) isEmpty() bool {
+func (h *HTTPHealthCheckArgs) IsZero() bool {
```

Comment:
Can we add a doc comment here saying that it implements the [`yaml.IsZeroer`](https://pkg.go.dev/gopkg.in/yaml.v3#IsZeroer)  interface? 🥺